miden/src/tools/mod.rs

```diff
@@ -62,7 +62,7 @@ impl ProgramInfo {
 
 /// Returns program analysis of a given program.
 pub fn analyze(program: &str, inputs: ProgramInputs) -> Result<ProgramInfo, ProgramError> {
-    let assembler = Assembler::new();
+    let assembler = Assembler::default();
```

Comment:
Shouldn't the assembler be instantiated in debug mode here?

Reply:
Thanks. Fixed.